### PR TITLE
Text disclaimer fixes

### PR DESCRIPTION
**Pull request by anasufana**
- Re-added text format & font color
- Added text size options instead of px size for the disclaimer text

### CODE REVIEW

File: src/components/__snapshots__/text-disclaimer.test.jsx.snap [hunk 1]
```diff
@@ -13,10 +13,10 @@ exports[`<TextDisclaimer/> clicking the read more button should make the renderi
     }
   >
     <h3
+      className="font_style__H1"
       style={
         Object {
           "color": "#2897FF",
-          "fontSize": "34px",
```
Comment (eugene-matvejev) [inline src/components/__snapshots__/text-disclaimer.test.jsx.snap]: 👍

File: src/components/__snapshots__/text-disclaimer.test.jsx.snap [hunk 2]
```diff
@@ -133,7 +133,7 @@ exports[`<TextDisclaimer/> render with default/required props 1`] = `
 </div>
 `;
 
-exports[`<TextDisclaimer/> rendering with optional props proprty = color : red 1`] = `
+exports[`<TextDisclaimer/> rendering with optional props property = color : red 1`] = `
```
Comment (eugene-matvejev) [inline src/components/__snapshots__/text-disclaimer.test.jsx.snap]: 👍

File: src/components/text-disclaimer.test.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ describe('<TextDisclaimer/>', () => {
     beforeEach(() => {
         props = {
             title: 'Banana',
-            fontSize: '34',
+            fontSize: 'H1',
```
Comment (eugene-matvejev) [inline src/components/text-disclaimer.test.jsx]: it is not a dead one?

Comment (anasufana) [reply src/components/text-disclaimer.test.jsx]: do you mean the `beforeEach`

**Pull request merged**